programs/drift/src/controller/orders.rs

```diff
@@ -1339,6 +1328,20 @@ fn fulfill_perp_order(
         emit!(order_record)
     }
 
+    let maker_risk_reducing = if let Some(maker) = maker {
```

Comment:
wonder if checking before/after reduce allows for some intermediate risk increasing orders to go through..

Reply:
shouldn't matter as long as the final state is lower risk, it just affects the margin requirement check we enforce anyway